tests/test_topology.py

```diff
@@ -327,3 +327,19 @@ def test_topology_join_keep_resSeq(get_fn):
 
     eq(out_resSeq_keepId_True, expected_resSeq_keepId_True)
     eq(out_resSeq_keepId_False, expected_resSeq_keepId_False)
+
+def test_preserve_chain_id(get_fn): 
+    # This pdb has three chains, A, B and C.  
+    traj = md.load(get_fn("1vii_sustiva_water.pdb"))
+    sub_traj = traj.atom_slice(traj.top.select('chainid 1'))
+    eq(sub_traj.top._chains[0].chain_id, 'B')
```

Comment:
This seems weird but is probably correct, could you verify my understanding?

* 'chainid 1' grabs the 1-th (second) chain
* this chain (1 residue long) is the ligand: https://github.com/mdtraj/mdtraj/blob/36b4cee61d936d9ecee875fbdde37120b59a4d51/tests/data/1vii_sustiva_water.pdb#L601-L631
* the "chain ID" of this chain happens to be `B`

I think I'm just getting confused by the difference between a chain _index_ and a chain _ID_ ...


```python
In [1]: import openmm.app

In [2]: {
   ...:     (chain.index, chain.id)
   ...:     for chain in [
   ...:         residue.chain
   ...:         for residue in openmm.app.PDBFile("1vii_sustiva_water.pdb").topology.residues()
   ...:     ]
   ...: }
Out[2]: {(0, 'A'), (1, 'B'), (2, 'C')}

Reply:
Also, what's the TER record if written from this subset using this patch?

Reply:
Sorry - I completely forgot about this.  I will attend to this this week!